config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show"
+  post "/guests/:id", to: "guests#create"
```

Comment:
Following up on a note that I had written earlier in your `GuestsController`, let's have this route go to a `guest_rooms` controller. Currently, this route is a RESTful route for the creation of a guest resource, so let's leave that route available in case we ever need to add in that functionality. One thing we could do instead is:

```ruby
post '/guests/:id/rooms', to:'guest_rooms#create'
```